working/manual_checks_after_generic.sql

```diff
@@ -1,51 +1,143 @@
  from concept a
-left join concept_relationship r on a.concept_id= r.concept_id_1 and r.invalid_reason is null and r.relationship_Id ='Maps to'  
+left join concept_relationship r on a.concept_id= r.concept_id_1 and r.invalid_reason is null and r.relationship_Id ='Maps to'
 left join concept  b on b.concept_id = r.concept_id_2
 left join devv5.concept  c on c.concept_id = a.concept_id
-where a.vocabulary_id ='<your_vocab>' 
+where a.vocabulary_id ='<your_vocab>'
 and c.concept_id is null and b.concept_id is null
 ;
---looking at new concepts and their mapping -- mapping present
+
+--04. looking at new concepts and their mapping -- mapping present
 select a.concept_code, a.concept_name, a.domain_id , b.concept_name, b.vocabulary_id
  from concept a
-  join concept_relationship r on a.concept_id= r.concept_id_1 and r.invalid_reason is null and r.relationship_Id ='Maps to'  
+  join concept_relationship r on a.concept_id= r.concept_id_1 and r.invalid_reason is null and r.relationship_Id ='Maps to'
  join concept  b on b.concept_id = r.concept_id_2
 left join devv5.concept  c on c.concept_id = a.concept_id
-where a.vocabulary_id ='<your_vocab>' 
+where a.vocabulary_id ='<your_vocab>'
 and c.concept_id is null  
 ;
---concepts changed their mapping
-with new_map as (
-select a.concept_code, a.concept_name, string_agg (r.relationship_id, '-' order by b.concept_code ) as relationship_agg , string_agg (b.concept_code, '-' order by b.concept_code ) as code_agg ,  string_agg (b.concept_name, '-/-' order by b.concept_code) as name_agg from concept a
- join concept_relationship r on a.concept_id = concept_id_1 and r.relationship_id in ('Maps to', 'Maps to value') and r.invalid_reason is null
- join concept b on b.concept_id = concept_id_2 
-where a.vocabulary_id= '<your_vocab>'  and a.invalid_reason is null  
 
-group by a.concept_code, a.concept_name 
+--05. looking at new concepts and their ancestry -- mapping absent
+select a.concept_code, a.concept_name, a.concept_class_id, a.domain_id, b.concept_name, b.concept_class_id, b.vocabulary_id
+from concept a
+left join concept_relationship r on a.concept_id= r.concept_id_1 and r.invalid_reason is null and r.relationship_Id ='Is a'
+left join concept b on b.concept_id = r.concept_id_2
+left join devv5.concept  c on c.concept_id = a.concept_id
+where a.vocabulary_id ='<your_vocab>'
+and c.concept_id is null and b.concept_id is null
+;
+
+--06. looking at new concepts and their ancestry -- mapping present
+select a.concept_code, a.concept_name, a.concept_class_id, a.domain_id, b.concept_name, b.concept_class_id, b.vocabulary_id
+from concept a
+join concept_relationship r on a.concept_id= r.concept_id_1 and r.invalid_reason is null and r.relationship_Id ='Is a'
+join concept  b on b.concept_id = r.concept_id_2
+left join devv5.concept  c on c.concept_id = a.concept_id
+where a.vocabulary_id ='<your_vocab>'
+and c.concept_id is null
+;
+
+--07. concepts changed their mapping

```

Comment:
@Alexdavv did you changed anything here? it shows like a big diff

Reply:
no